.github/workflows/actions.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: [2.6.8, 2.5.9]
+        ruby-version: [2.6.8]
```

Comment:
At the moment ZAS specifies `s.required_ruby_version = '>= 2.0'`. Should we bump the ruby version requirement if we are not testing it against older ruby versions?